tests/bench/test_bench_cli.py

```
@@ -0,0 +1,68 @@
+from logging import getLogger
+
+import pytest
+
+from sema.bench.__main__ import _main, get_arg_parser
+
+log = getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "args_line, expected",
```

Ruff

Wrong type passed to first argument of `@pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)